components/ResponsiveImage/index.tsx

```
@@ -0,0 +1,20 @@
+    image: string;
+    useOutline?: boolean;
+}
+
+const ResponsiveImage: React.FC<ResponsiveImageProps> = ({ isMobile, image, useOutline = true }) => {
+    const size = isMobile ? "150px" : "300px";
+    const outlineClass = useOutline ? "outline-dashed outline-offset-8 outline-cyan-300" : "";
+    const className = isMobile
+        ? `overflow-hidden object-cover object-center aspect-square rounded-full ${outlineClass} h-36 w-36`
+        : `overflow-hidden object-cover object-center aspect-square rounded-full ${outlineClass} h-[300] w-[350]`;
+
+    {/* TODO: replace with Next/image */ }
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)